counter/src/Main.purs

```diff
@@ -1,25 +1,25 @@
-module Main where
-
-import Prelude
-
-import Counter.Counter as Counter
-import Data.Function.Uncurried (runFn2)
-import Data.Maybe (Maybe(..))
-import Effect (Effect)
-import Effect.Console (log)
-import Elmish as Elmish
-import Elmish.React as React
-import Web.DOM.NonElementParentNode (getElementById)
-import Web.HTML (window)
-import Web.HTML.HTMLDocument (toNonElementParentNode)
-import Web.HTML.Window (document)
-
-main :: Effect Unit
-main = do
-  el <- getElementById "app" =<< (map toNonElementParentNode <<< document) =<< window
-  case el of
-    Nothing -> log "Can't find #app element"
-    Just e -> runFn2 React.reactMount e =<< Elmish.construct Counter.def onError
-
-  where
-    onError = log
\ No newline at end of file
+module Main where
+
+import Prelude
+
+import Counter.Counter as Counter
+import Data.Function.Uncurried (runFn2)
+import Data.Maybe (Maybe(..))
+import Effect (Effect)
+import Effect.Console (log)
+import Elmish as Elmish
+import Elmish.React as React
+import Web.DOM.NonElementParentNode (getElementById)
+import Web.HTML (window)
+import Web.HTML.HTMLDocument (toNonElementParentNode)
+import Web.HTML.Window (document)
+
+main :: Effect Unit
+main = do
+  el <- getElementById "app" =<< (map toNonElementParentNode <<< document) =<< window
```

Comment:
I had some ideas on how to bring this into Elmish — this is kind of ridiculous 😉

Reply:
Yes, I was going to do that. This bootstrapping thing should really be part of Elmish.